tests/Unit/Policies/FeedbackPolicyTest.php

```diff
@@ -40,13 +40,7 @@ public function data_provider_for_all_permissions_as_creator()
     {
         return [
             'view-true' => [ 'view', self::createPermissionedUser('view feedbacks', true), true ],
-            'view-false' => [ 'view', self::createPermissionedUser('view feedbacks', false), false ],
```

Comment:
Since I'm assuming Feedback should only be viewed, would any of these checks be required?